### PR TITLE
Gecko fixes

### PR DESCRIPTION
**Pull request by poka-IT**

### CODE REVIEW

**Comment by poka-IT** [comment]
@RomeroYang are you OK to merge this MR ?

File: js_api/src/service/keyring.ts [hunk 1]
```diff
@@ -470,7 +469,7 @@ async function signBytesAsExtension(password: string, json: any) {
       }
       keyPair.decodePkcs8(password);
       resolve({
-        signature: u8aToHex(keyPair.sign(wrapBytes(json["data"]))),
+        signature: u8aToHex(keyPair.sign(unwrapBytes(json["data"]))),
```
Comment (RomeroYang) [inline js_api/src/service/keyring.ts:472]: I think we should use `wrapBytes` here according to [polkadot-js/extension](https://github.com/polkadot-js/extension/blob/master/packages/extension-base/src/background/RequestBytesSign.ts), so the signer will have the same behavior with the extension.

Comment (poka-IT) [reply js_api/src/service/keyring.ts:472]: Ok, but I need to unwrap one of my signature here: https://git.duniter.org/clients/gecko/-/blob/master/lib/providers/substrate_sdk.dart#L109-120

So should I add a new methode `signAsExtensionUnwrap` ?

Comment (RomeroYang) [reply js_api/src/service/keyring.ts:472]: where do you use the unwrapped signature? maybe you should [add message wrapping support](https://github.com/polkadot-js/apps/pull/6144) while do signatureVerify.

Comment (RomeroYang) [reply js_api/src/service/keyring.ts:472]: I mean you can add the wrapping both in sign&verify, then you will not need the unwrap one.

**Pull request closed without merging**